src/Components/ManageCard/ManageCard.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import { getCards } from './actions';
+
+export default class ManageCard extends Component {
+  constructor(props) {
+    super(props);
+    this.selectCard = this.selectCard.bind(this);
+  }
+
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(getCards());
+  }
+
+  selectCard(e) { 
+    const { dispatch, cards } = this.props;
+    let card = cards.find(card => card.id === e.target.dataset.cardid);
+    card = {...card, used: true  }
+    dispatch(serverSetActiveCard(card));
+  }
+
+  deleteCard() {
+    const { dispatch } = this.props;
+    console.log('make a delete function')
```

Comment:
Remove this `console.log` and the commented line below it

Reply:
Removed entire unused "deleteCard" function.